### ISSUE TITLE
CLI: Unexpected exception occurred while performing apply-configuration command

### ISSUE THREAD

**Issue by flederwiesel** [jenkinsci/configuration-as-code-plugin#1837]
### Jenkins and plugins versions report

<details>
  <summary>Environment</summary>

  
  ```text
Jenkins: 2.319.2
OS: Linux - 5.4.0-96-generic
---
ace-editor:1.1
active-directory:2.25.1
analysis-model-api:10.9.1
ant:1.13
antisamy-markup-formatter:2.7
apache-httpcomponents-client-4-api:4.5.13-1.0
authorize-project:1.4.0
bitbucket:214.v2fd4234d0554
bootstrap4-api:4.6.0-3
bootstrap5-api:5.1.3-4
bouncycastle-api:2.25
branch-api:2.7.0
build-name-setter:2.2.0
build-timeout:1.20
built-on-column:1.1
caffeine-api:2.9.2-29.v717aac953ff3
checks-api:1.7.2
clearcase:1.6.8
cloudbees-folder:6.17
cobertura:1.17
code-coverage-api:2.0.4
command-launcher:1.6
conditional-buildstep:1.4.1
config-file-provider:3.8.2
configuration-as-code:1.55.1
copyartifact:1.46.2
credentials:1074.v60e6c29b_b_44b_
credentials-binding:1.27.1
cvs:2.19
dashboard-view:2.18
data-tables-api:1.11.3-6
display-url-api:2.3.5
dtkit-api:3.0.0
durable-task:493.v195aefbb0ff2
echarts-api:5.2.2-2
email-ext:2.86
emailext-template:1.4
embeddable-build-status:2.0.3
envinject:2.4.0
envinject-api:1.8
external-monitor-job:1.7
font-awesome-api:5.15.4-5
forensics-api:1.7.0
git:4.10.3
git-client:3.11.0
git-parameter:0.9.15
git-server:1.10
github:1.34.1
github-api:1.301-378.v9807bd746da5
github-branch-source:2.11.4
gitlab-plugin:1.5.27
gradle:1.38
handlebars:3.0.8
htmlpublisher:1.28
jackson2-api:2.13.1-246.va8a9f3eaf46a
javadoc:1.6
javax-activation-api:1.2.0-2
javax-mail-api:1.6.2-5
jaxb:2.3.0.1
jdk-tool:1.5
jenkins-multijob-plugin:1.36
jersey2-api:2.35-3
jjwt-api:0.11.2-9.c8b45b8bb173
jquery:1.12.4-1
jquery3-api:3.6.0-2
jsch:0.1.55.2
junit:1.53
ldap:2.7
locale:1.4
lockable-resources:2.13
mailer:408.vd726a_1130320
mapdb-api:1.0.9.0
matrix-auth:3.0
matrix-project:1.20
maven-plugin:3.16
mercurial:2.16
momentjs:1.1.1
msbuild:1.30
nodejs:1.5.1
okhttp-api:4.9.3-105.vb96869f8ac3a
p4:1.12.0
pam-auth:1.6.1
parameterized-trigger:2.43
pipeline-build-step:2.15
pipeline-github-lib:1.0
pipeline-graph-analysis:188.v3a01e7973f2c
pipeline-input-step:446.vf27b_0b_83500e
pipeline-milestone-step:1.3.2
pipeline-model-api:1.9.3
pipeline-model-definition:1.9.3
pipeline-model-extensions:1.9.3
pipeline-rest-api:2.20
pipeline-stage-step:291.vf0a8a7aeeb50
pipeline-stage-tags-metadata:1.9.3
pipeline-stage-view:2.20
plain-credentials:1.7
plugin-util-api:2.12.0
popper-api:1.16.1-2
popper2-api:2.11.2-1
prism-api:1.25.0-2
rebuild:1.33
repo:1.14.0
resource-disposer:0.17
role-strategy:3.2.0
run-condition:1.5
scm-api:595.vd5a_df5eb_0e39
script-security:1131.v8b_b_5eda_c328e
snakeyaml-api:1.29.1
ssh:2.6.1
ssh-agent:1.24.1
ssh-credentials:1.19
ssh-slaves:1.33.0
sshd:3.1.0
structs:308.v852b473a2b8c
subversion:2.15.2
throttle-concurrents:2.6
timestamper:1.16
token-macro:267.vcdaea6462991
trilead-api:1.0.13
warnings-ng:9.11.0
windows-slaves:1.8
workflow-aggregator:2.6
workflow-api:1136.v7f5f1759dc16
workflow-basic-steps:2.24
workflow-cps:2648.va9433432b33c
workflow-cps-global-lib:552.vd9cc05b8a2e1
workflow-durable-task-step:1121.va_65b_d2701486
workflow-job:1145.v7f2433caa07f
workflow-multibranch:706.vd43c65dec013
workflow-scm-step:2.13
workflow-step-api:622.vb_8e7c15b_c95a_
workflow-support:813.vb_d7c3d2984a_0
ws-cleanup:0.40
xunit:3.0.5
  ```

</details>

### What Operating System are you using (both controller, and any agents involved in the problem)?

Latest docker LTS image jenkins/jenkins:lts (in my opinion...) started as container from systemd.

```
$ docker container ls -a
CONTAINER ID   IMAGE                 COMMAND                  CREATED          STATUS          PORTS                                                  NAMES
3df790c80eb0   jenkins/jenkins:lts   "/sbin/tini -- /usr/…"   51 minutes ago   Up 51 minutes   0.0.0.0:8080->8080/tcp, :::8080->8080/tcp, 50000/tcp   jenkins
$ docker pull jenkins/jenkins:lts
lts: Pulling from jenkins/jenkins
Digest: sha256:983890f538ff42b2d153e0b5f45d97e750e80f08238ddb2bcf5c5dc4a606e77f
Status: Image is up to date for jenkins/jenkins:lts
docker.io/jenkins/jenkins:lts
$ docker image ls -a
REPOSITORY        TAG       IMAGE ID       CREATED      SIZE
jenkins/jenkins   lts       9aee0d53624f   9 days ago   441MB
```

### Reproduction steps

- Setup clean Jenkins instance from docker image jenkins/jenkins:lts
  - Add user/group "jenkins" to run the service
  - Add user "jenkins" to "docker" group
  - Start Jenkins docker
  - Download `jenkins-cli.jar`
  - Install plugins
  - **Restart** Jenkins
  - You may use this script -- tested on Ubuntu-20.04 host:
  ```
  #!/bin/bash
  
  if [[ "$EUID" != "0" ]]; then
          echo "Script must be run as root." >&2
          exit 1
  fi
  
  readonly HOME=$(eval echo ~$(printf %q "$SUDO_USER"))
  readonly scriptdir=$(dirname "$(realpath "$0")")
  
  deps=(default-jre docker.io)
  
  for d in "${deps[@]}"
  do
          dpkg --list "$d" | grep -q '^ii' || install+=("$d")
  done
  
  if [[ ${install[@]} ]]; then
          # Install necessary binaries
          # The docker.io install callback creates the docker group
  
          apt update
          apt upgrade --yes
          apt install --yes "${install[@]}"
  fi
  
  getent group jenkins &>/dev/null ||
  groupadd --system jenkins
  
  getent passwd jenkins &>/dev/null ||
  useradd --system -g jenkins -s /sbin/nologin jenkins
  
  gid=$(getent group docker)
  
  id jenkins | grep -q "groups=.*\<$gid\>" ||
  usermod -aG docker jenkins
  
  [[ -e /var/lib/jenkins ]] &&
  {
          echo "Refusing to use /var/lib/jenkins, as it already exist." >&2
          exit 2
  }
  
  mkdir -p /var/lib/jenkins
  
  # The user in the docker image has uid 1000.
  # We are going to mount this dir into the docker container, so it must be
  # accessible to UID 1000...
  chown 1000:jenkins /var/lib/jenkins
  
  ###
  
  docker pull jenkins/jenkins:lts
  
  # Setup systemd
  cat <<"EOF" > /lib/systemd/system/jenkins.service
  [Unit]
  Description=Jenkins Server
  Documentation=https://jenkins.io/doc/
  After=docker.service
  Requires=docker.service
  
  [Service]
  Type=simple
  User=jenkins
  Group=jenkins
  SyslogIdentifier=jenkins
  TimeoutStartSec=0
  Restart=on-failure
  RestartSec=30s
  ExecStart=/usr/bin/docker run \
          --rm \
          --name jenkins \
          --publish 8080:8080 \
          --volume /var/lib/jenkins:/var/jenkins_home \
          jenkins/jenkins:lts
  ExecStop=/usr/bin/docker stop jenkins
  
  [Install]
  WantedBy=multi-user.target
  EOF
  
  systemctl daemon-reload
  systemctl enable jenkins
  systemctl start  jenkins
  
  # Wait for Jenkins being up and running
  sleep 30
  
  passwd=$(docker exec jenkins cat /var/jenkins_home/secrets/initialAdminPassword 2>/dev/null)
  echo "admin:$passwd" | tee $HOME/.jenkins-auth
  
  # Get CLI
  
  [ -e /usr/share/java/jenkins-cli.jar ] ||
  wget -P /usr/share/java http://localhost:8080/jnlpJars/jenkins-cli.jar
  
  jenkins-cli()
  {
          java -jar /usr/share/java/jenkins-cli.jar \
                  -s http://localhost:8080/ -auth \
                  @$HOME/.jenkins-auth "$@"
  }
  
  # Initial Jenkins setup
  
  jenkins-cli install-plugin $(cat <<"EOF"
  active-directory
  ant
  antisamy-markup-formatter
  authorize-project
  bitbucket
  build-name-setter
  build-timeout
  clearcase
  cloudbees-folder
  cobertura
  conditional-buildstep
  config-file-provider
  configuration-as-code
  copyartifact
  credentials-binding
  cvs
  dashboard-view
  email-ext
  emailext-template
  embeddable-build-status
  git
  github
  github-branch-source
  gitlab-plugin
  git-parameter
  gradle
  htmlpublisher
  jenkins-multijob-plugin
  junit
  ldap
  locale
  mailer
  matrix-auth
  matrix-project
  msbuild
  nodejs
  p4
  pam-auth
  parameterized-trigger
  pipeline-github-lib
  pipeline-stage-view
  rebuild
  repo
  role-strategy
  ssh
  ssh-agent
  ssh-slaves
  subversion
  throttle-concurrents
  timestamper
  warnings-ng
  windows-slaves
  workflow-aggregator
  ws-cleanup
  xunit
  EOF
  )
  
  jenkins-cli restart
  
  # Wait for Jenkins being back up
  sleep 45
    ```
- Export config via `jenkins-cli.jar`
```
java -jar /usr/share/java/jenkins-cli.jar -s http://localhost:8080/ -auth @$HOME/jenkins-auth export-configuration > config.yml
```

- Import config via `jenkins-cli.jar`
```
java -jar /usr/share/java/jenkins-cli.jar -s http://localhost:8080/ -auth @$HOME/jenkins-auth apply-configuration < config.yml
```

### Expected Results

No exception.

### Actual Results

```
ERROR: Unexpected exception occurred while performing apply-configuration command.
java.lang.NullPointerException
        at hudson.model.MyViewsProperty.save(MyViewsProperty.java:139)
        at hudson.BulkChange.commit(BulkChange.java:97)
        at io.jenkins.plugins.casc.BaseConfigurator.configure(BaseConfigurator.java:270)
        at io.jenkins.plugins.casc.impl.configurators.DataBoundConfigurator.configure(DataBoundConfigurator.java:82)
        at io.jenkins.plugins.casc.impl.configurators.HeteroDescribableConfigurator.lambda$doConfigure$16668e2$1(HeteroDescribableConfigurator.java:277)
        at io.vavr.CheckedFunction0.lambda$unchecked$52349c75$1(CheckedFunction0.java:247)
        at io.jenkins.plugins.casc.impl.configurators.HeteroDescribableConfigurator.doConfigure(HeteroDescribableConfigurator.java:277)
        at io.jenkins.plugins.casc.impl.configurators.HeteroDescribableConfigurator.lambda$null$2(HeteroDescribableConfigurator.java:86)
        at io.vavr.control.Option.map(Option.java:392)
        at io.jenkins.plugins.casc.impl.configurators.HeteroDescribableConfigurator.lambda$configure$3(HeteroDescribableConfigurator.java:86)
        at io.vavr.Tuple2.apply(Tuple2.java:238)
        at io.jenkins.plugins.casc.impl.configurators.HeteroDescribableConfigurator.configure(HeteroDescribableConfigurator.java:83)
        at io.jenkins.plugins.casc.impl.configurators.HeteroDescribableConfigurator.configure(HeteroDescribableConfigurator.java:55)
        at io.jenkins.plugins.casc.BaseConfigurator.configure(BaseConfigurator.java:344)
        at io.jenkins.plugins.casc.BaseConfigurator.configure(BaseConfigurator.java:275)
        at io.jenkins.plugins.casc.impl.configurators.DataBoundConfigurator.configure(DataBoundConfigurator.java:82)
        at io.jenkins.plugins.casc.BaseConfigurator.configure(BaseConfigurator.java:344)
        at io.jenkins.plugins.casc.BaseConfigurator.configure(BaseConfigurator.java:275)
        at io.jenkins.plugins.casc.impl.configurators.DataBoundConfigurator.configure(DataBoundConfigurator.java:82)
        at io.jenkins.plugins.casc.impl.configurators.HeteroDescribableConfigurator.lambda$doConfigure$16668e2$1(HeteroDescribableConfigurator.java:277)
        at io.vavr.CheckedFunction0.lambda$unchecked$52349c75$1(CheckedFunction0.java:247)
        at io.jenkins.plugins.casc.impl.configurators.HeteroDescribableConfigurator.doConfigure(HeteroDescribableConfigurator.java:277)
        at io.jenkins.plugins.casc.impl.configurators.HeteroDescribableConfigurator.lambda$null$2(HeteroDescribableConfigurator.java:86)
        at io.vavr.control.Option.map(Option.java:392)
        at io.jenkins.plugins.casc.impl.configurators.HeteroDescribableConfigurator.lambda$configure$3(HeteroDescribableConfigurator.java:86)
        at io.vavr.Tuple2.apply(Tuple2.java:238)
        at io.jenkins.plugins.casc.impl.configurators.HeteroDescribableConfigurator.configure(HeteroDescribableConfigurator.java:83)
        at io.jenkins.plugins.casc.impl.configurators.HeteroDescribableConfigurator.check(HeteroDescribableConfigurator.java:92)
        at io.jenkins.plugins.casc.impl.configurators.HeteroDescribableConfigurator.check(HeteroDescribableConfigurator.java:55)
        at io.jenkins.plugins.casc.BaseConfigurator.configure(BaseConfigurator.java:350)
        at io.jenkins.plugins.casc.BaseConfigurator.check(BaseConfigurator.java:286)
        at io.jenkins.plugins.casc.ConfigurationAsCode.lambda$checkWith$8(ConfigurationAsCode.java:776)
        at io.jenkins.plugins.casc.ConfigurationAsCode.invokeWith(ConfigurationAsCode.java:712)
        at io.jenkins.plugins.casc.ConfigurationAsCode.checkWith(ConfigurationAsCode.java:776)
        at io.jenkins.plugins.casc.ConfigurationAsCode.configureWith(ConfigurationAsCode.java:761)
        at io.jenkins.plugins.casc.ConfigurationAsCode.configureWith(ConfigurationAsCode.java:637)
        at io.jenkins.plugins.casc.ConfigurationAsCode.configureWith(ConfigurationAsCode.java:631)
        at io.jenkins.plugins.casc.cli.ApplyConfigurationCommand.run(ApplyConfigurationCommand.java:30)
        at hudson.cli.CLICommand.main(CLICommand.java:249)
        at hudson.cli.CLIAction$ServerSideImpl.run(CLIAction.java:282)
        at hudson.cli.CLIAction$PlainCliEndpointResponse$1.run(CLIAction.java:313)
        at jenkins.util.FullDuplexHttpService.download(FullDuplexHttpService.java:118)
        at jenkins.util.FullDuplexHttpService$Response.generateResponse(FullDuplexHttpService.java:178)
        at org.kohsuke.stapler.HttpResponseRenderer$Default.handleHttpResponse(HttpResponseRenderer.java:124)
        at org.kohsuke.stapler.HttpResponseRenderer$Default.generateResponse(HttpResponseRenderer.java:69)
        at org.kohsuke.stapler.Function.renderResponse(Function.java:160)
        at org.kohsuke.stapler.Stapler.tryInvoke(Stapler.java:723)
        at org.kohsuke.stapler.Stapler.invoke(Stapler.java:898)
        at org.kohsuke.stapler.MetaClass$9.dispatch(MetaClass.java:457)
        at org.kohsuke.stapler.Stapler.tryInvoke(Stapler.java:766)
        at org.kohsuke.stapler.Stapler.invoke(Stapler.java:898)
        at org.kohsuke.stapler.Stapler.invoke(Stapler.java:694)
        at org.kohsuke.stapler.Stapler.service(Stapler.java:240)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
        at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:799)
        at org.eclipse.jetty.servlet.ServletHandler$ChainEnd.doFilter(ServletHandler.java:1626)
        at hudson.util.PluginServletFilter$1.doFilter(PluginServletFilter.java:156)
        at jenkins.security.ResourceDomainFilter.doFilter(ResourceDomainFilter.java:80)
        at hudson.util.PluginServletFilter$1.doFilter(PluginServletFilter.java:153)
        at jenkins.telemetry.impl.UserLanguages$AcceptLanguageFilter.doFilter(UserLanguages.java:128)
        at hudson.util.PluginServletFilter$1.doFilter(PluginServletFilter.java:153)
        at hudson.plugins.locale.LocaleFilter.doFilter(LocaleFilter.java:42)
        at hudson.util.PluginServletFilter$1.doFilter(PluginServletFilter.java:153)
        at hudson.util.PluginServletFilter.doFilter(PluginServletFilter.java:159)
        at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193)
        at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601)
        at hudson.cli.CliCrumbExclusion.process(CliCrumbExclusion.java:46)
        at hudson.security.csrf.CrumbFilter.doFilter(CrumbFilter.java:127)
        at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193)
        at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601)
        at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:92)
        at jenkins.security.AcegiSecurityExceptionFilter.doFilter(AcegiSecurityExceptionFilter.java:52)
        at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:97)
        at hudson.security.UnwrapSecurityExceptionFilter.doFilter(UnwrapSecurityExceptionFilter.java:53)
        at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:97)
        at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:121)
        at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:115)
        at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:97)
        at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:105)
        at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:97)
        at org.springframework.security.web.authentication.rememberme.RememberMeAuthenticationFilter.doFilter(RememberMeAuthenticationFilter.java:101)
        at org.springframework.security.web.authentication.rememberme.RememberMeAuthenticationFilter.doFilter(RememberMeAuthenticationFilter.java:92)
        at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:97)
        at org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:218)
        at org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:212)
        at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:97)
        at jenkins.security.BasicHeaderProcessor.success(BasicHeaderProcessor.java:139)
        at jenkins.security.BasicHeaderProcessor.doFilter(BasicHeaderProcessor.java:86)
        at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:97)
        at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:110)
        at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:80)
        at hudson.security.HttpSessionContextIntegrationFilter2.doFilter(HttpSessionContextIntegrationFilter2.java:62)
        at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:97)
        at hudson.security.ChainedServletFilter.doFilter(ChainedServletFilter.java:109)
        at hudson.security.HudsonFilter.doFilter(HudsonFilter.java:171)
        at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193)
        at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601)
        at org.kohsuke.stapler.compression.CompressionFilter.doFilter(CompressionFilter.java:51)
        at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193)
        at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601)
        at hudson.util.CharacterEncodingFilter.doFilter(CharacterEncodingFilter.java:85)
        at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193)
        at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601)
        at org.kohsuke.stapler.DiagnosticThreadNameFilter.doFilter(DiagnosticThreadNameFilter.java:30)
        at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193)
        at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601)
        at jenkins.security.SuspiciousRequestFilter.doFilter(SuspiciousRequestFilter.java:39)
        at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193)
        at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601)
        at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:548)
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
        at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:578)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:235)
        at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1624)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233)
        at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1434)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188)
        at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:501)
        at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1594)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)
        at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1349)
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
        at org.eclipse.jetty.server.Server.handle(Server.java:516)
        at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:388)
        at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:633)
        at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:380)
        at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:277)
        at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)
        at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:338)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:315)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
        at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:386)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
        at java.base/java.lang.Thread.run(Thread.java:829)
```

### Anything else?

The offending line is `- "myView"`:

```
jenkins:
  securityRealm:
    local:
      users:
      - id: "admin"
        properties:
        - "myView"
```

With this commented out, the import succeeds. As Jenkins newbie, I have never defined a view, so I cannot tell whether to explicitly defining a "myView" would help....

**Comment by timja** [comment]
That is provided by Jenkins core, it needs fixing at some point but it’s not a bug in jcasc

remove it from your config to fix the error

**Comment by timja** [comment]
Jenkins core issue: https://issues.jenkins.io/browse/JENKINS-61985

**Comment by basil** [comment]
Fixed in https://github.com/jenkinsci/jenkins/pull/6202. Released in 2.344.
